lara-typescript/src/interactive-api-client/api.ts

```diff
@@ -396,14 +396,13 @@ type WriteAttachmentParams = Omit<IWriteAttachmentRequest, "requestId" | "operat
 export const writeAttachment = (params: WriteAttachmentParams): Promise<Response> => {
   return new Promise<Response>((resolve, reject) => {
     const client = getClient();
-    const { content, contentType, options, ...others } = params;
+    const { content, contentType, options = {}, ...others } = params;
     const request: IAttachmentUrlRequest = { ...others, operation: "write", requestId: client.getNextRequestId() };
```

Comment:
Shouldn't the contentType be passed in the request object?
It is needed to create the presigned URL.

Reply:
🤦 Fixed